datafusion/core/src/dataframe.rs

```diff
@@ -505,17 +505,40 @@ impl DataFrame {
         self.plan.schema()
     }
 
-    /// Return the unoptimized logical plan represented by this DataFrame.
+    /// Return the unoptimized logical plan
+    pub fn logical_plan(&self) -> &LogicalPlan {
+        &self.plan
+    }
+
+    /// Return the logical plan represented by this DataFrame without running the optimizers
```

Comment:
👍 
I wonder if we should call it `into_unoptimized_plan` for consistency 🤔

Reply:
Done